src/commands/run.py

```diff
@@ -113,3 +123,22 @@ def remove_done_tasks():
             session.commit()
             Session.remove()
             run_print('Done')
+
+    def get_confirmation(configs, nodes):
+        click.echo("""
+You are about to run: {}\nOver nodes: {}\n
+""".strip().format(', '.join([c.name() for c in configs]),
+                   ', '.join(nodes)))
+        question = "Please enter [y/n] or 'more' for more info"
+        while "answer is invalid":
+            reply = click.prompt(question)
+            if reply == 'y':
+                return True
+            if reply == 'n':
+                return False
+            if reply == 'more':
+                for config in configs:
+                    click.echo("\nTool: {}".format(config.name()) + \
```

Comment:
Multi-line comments shouldn't be joined in this manner. Instead a heredoc should be used:
https://stackoverflow.com/a/10112665

Reply:
it's been removed